### PR TITLE
Sendgrid: Fix an error trying to assign to an undefined object when using the Reply To parameters

### PR DESCRIPTION
**Pull request by aca02djr**

### CODE REVIEW

**Comment by dylburger** [comment]
Thanks for submitting this PR! When we review PRs, we follow the [Pipedream component guidelines](https://pipedream.com/docs/components/guidelines/). If you're not familiar, here's a quick checklist:

- [ ] [Create components to address specific use cases whenever possible](https://pipedream.com/docs/components/guidelines/#component-scope)
- [ ] [Component `key`s should follow the format `app_name_slug-slugified-component-name`](https://pipedream.com/docs/components/guidelines/#component-key-pattern)
- [ ] Components should follow the [standard directory structure](https://pipedream.com/docs/components/guidelines/#folder-structure)
- [ ] [Prefer Node.js client libraries to REST APIs](https://pipedream.com/docs/components/guidelines/#using-apis-vs-client-libraries)
- [ ] When making API requests, [handle pagination](https://pipedream.com/docs/components/guidelines/#pagination) to ensure all data / events are processed
- [ ] [Use `secret` props](https://pipedream.com/docs/components/guidelines/#capturing-sensitive-data) to capture sensitive data
- [ ] Props and methods should be defined in [app files](https://pipedream.com/docs/components/guidelines/#app-files) whenever possible
- [ ] Document methods with [JS Docs](https://pipedream.com/docs/components/guidelines/#methods)
- [ ] [Use `optional` props](https://pipedream.com/docs/components/guidelines/#optional-vs-required-props) whenever possible, and [set a `default` value](https://pipedream.com/docs/components/guidelines/#default-values) where you can
- [ ] Use [async options](https://pipedream.com/docs/components/guidelines/#async-options) to accept user input wherever possible

File: components/sendgrid/actions/send-email-multiple-recipients/send-email-multiple-recipients.mjs [hunk 1]
```diff
@@ -173,7 +173,7 @@ export default {
     //specified, with `email` being required.
     let replyTo = undefined;
     if (this.replyToEmail) {
-      replyTo.email = this.replyToEmail;
+      replyTo = { email: this.replyToEmail };
```
Comment (andrewjschuang) [inline components/sendgrid/actions/send-email-multiple-recipients/send-email-multiple-recipients.mjs:176]:
```suggestion
      replyTo = {
        email: this.replyToEmail,
      };
```

File: components/sendgrid/actions/send-email-single-recipient/send-email-single-recipient.mjs [hunk 2]
```diff
@@ -251,7 +251,7 @@ export default {
     //specified, with `email` being required.
     let replyTo = undefined;
     if (this.replyToEmail) {
-      replyTo.email = this.replyToEmail;
+      replyTo = { email: this.replyToEmail };
```
Comment (andrewjschuang) [inline components/sendgrid/actions/send-email-single-recipient/send-email-single-recipient.mjs:254]:
```suggestion
      replyTo = {
        email: this.replyToEmail,
      };
```

**Review by andrewjschuang — CHANGES_REQUESTED** [review]
@aca02djr thanks for the contribution!

Just need to change formatting for ESLint

**Comment by vunguyenhung** [comment]
Since the user doesn't have write permission, I will merge this PR on behave of him 🙏

**Pull request merged**